api.raml

```diff
@@ -320,8 +330,7 @@ schemas:
       responses:
         200:
           description:
```

Comment:
I'm not positive, but I think this `description:` will need to be `description: |` since the text is on the next line.